packages/core/src/payment/payment.ts

```diff
@@ -212,3 +212,16 @@ export interface FormattedVaultedInstrument {
 export interface FormattedPayload<T> {
     formattedPayload: T;
 }
+
+export interface StripeV3FormattedPayload {
+    credit_card_token?: {
```

Comment:
Are all these supposed to be undefined/optionals?

Reply:
Yes, because if we take a look at `stripev3-payment-strategy.ts` where we are using the `StripeV3FormattedPayload`
some of the variables not have been using every time so we keep them optional whenever we call the `formattedPayload` with that variable.